uast/uast.go

```diff
@@ -31,316 +31,280 @@ const (
-	// Depending on the language his child node representing the actual value could be some kind or
-	// literal or even expressions that can resolved at runtime (if interpreted) or compile time.
-	FunctionDeclarationArgumentDefaultValue
-	// FunctionDeclarationVarArgsList is the node representing whatever syntax the language has to
-	// indicate that from that point in the argument list the function can get a variable number
-	// of arguments (e.g. "..." in C-ish languages, "Object..." in Java, "*args" in Python, etc).
-	FunctionDeclarationVarArgsList
-
-	// TypeDeclaration is the declaration of a type. It could be a class or
-	// interface in Java, a struct, interface or alias in Go, etc. Except for pure or forward declarations
-	// it will usually have a TypeDeclarationBody child and for OOP languages a TypeDeclarationBases and/or
-	// TypeDeclarationInterfaces.
-	TypeDeclaration
-	TypeDeclarationBody
-	// TypeDeclarationBases are the Types that the current inherits from in OOP languages.
-	TypeDeclarationBases
-	// TypeDeclarationImplements are the Types (usually interfaces) that the Type implements.
-	TypeDeclarationImplements
-
-	// VisibleFromInstance marks modifiers that declare visibility from instance.
-	VisibleFromInstance
-	// VisibleFromType marks modifiers that declare visibility from the same
-	// type (e.g. class, trait).
-	// Implies VisibleFromInstance.
-	VisibleFromType
-	// VisibleFromSubtype marks modifiers that declare visibility from
-	// subtypes (e.g. subclasses).
-	// Implies VisibleFromInstance and VisibleFromType.
-	VisibleFromSubtype
-	// VisibleFromSubpackage marks modifiers that declare visibility from the
-	// same package.
-	VisibleFromPackage
-	// VisibleFromSubpackage marks modifiers that declare visibility from
-	// subpackages.
-	// Implies VisibleFromInstance, VisibleFromType and VisibleFromPackage.
-	VisibleFromSubpackage
-	// VisibleFromModule marks modifiers that declare visibility from the
-	// same module (e.g. Java JAR).
-	// Implies VisibleFromInstance and VisibleFromType.
-	VisibleFromModule
-	// VisibleFromFriend marks modifiers that declare visibility from friends
-	// (e.g. C++ friends).
-	// Implies VisibleFromInstance and VisibleFromType.
-	VisibleFromFriend
-	// VisibleFromWorld implies full public visibility. Implies all other
-	// visibility levels.
-	VisibleFromWorld
+	// Add is an arithmetic operator (i.e. `+`).
+	Add
+
+	// Substract in an arithmetic operator (i.e. `-`).
+	Substract
+
+	// Multiply is an arithmetic operator (i.e. `*`).
+	Multiply
+
+	// Divide is an arithmetic operator (i.e. `/`).
+	Divide
+
+	// Modulo is an arithmetic operator (i.e. `%`, `mod`, etc.).
+	Modulo
+
+	// Package indicates that a package level property
+	Package
+
+	// Declaration is a construct to specify properties of an identifier
+	Declaration
+
+	// Import indicates an import level property
+	Import
+
+	// Path is a qualified name of some construct
+	Path
```

Comment:
By the compiler warnings, this could be being overwrited by the `Path` in line 520.